packages/acceptance-tests/pre_packaging

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+# This pre_packaging script is required to do the `go mod vendor` while we have
+# internet access; unfortunately, this means we run it on the developer
+# workstation (or, as the case may be, CI).
+
+set -o errexit -o nounset
+
+export GOPATH="${BUILD_DIR}"
+export GOCACHE="${BUILD_DIR}/.go-cache"
```

Comment:
Need to be hermetic, mostly for CI.  Inspired by [eirini-bosh-release](https://github.com/cloudfoundry-community/eirini-bosh-release/blob/master/packages/eirini-loggregator-bridge/pre_packaging).